src/main/java/com/puppycrawl/tools/checkstyle/Checker.java

```diff
@@ -213,28 +213,22 @@ public int process(List<File> files) throws CheckstyleException {
         if (cacheFile != null) {
             cacheFile.putExternalResources(getExternalResourceLocations());
         }
-
-        // Prepare to start
+        // prepare
         fireAuditStarted();
         for (final FileSetCheck fsc : fileSetChecks) {
             fsc.beginProcessing(charset);
         }
-
-        final List<File> targetFiles = files.stream()
+        // process
+        processFiles(
+            files
+                .stream()
                 .filter(file -> CommonUtil.matchesFileExtension(file, fileExtensions))
-                .collect(Collectors.toUnmodifiableList());
-        processFiles(targetFiles);
-
-        // Finish up
-        // It may also log!!!
+                .collect(Collectors.toUnmodifiableList()));
+        // complete - it may also log!!!
```

Comment:
```suggestion
        // finish up - it may also log!!!
```
might be more concrete.
Initially idea was to keep it kiss and follow the `one-word` pattern to simplify the definition.